src/tools/compiletest/src/header.rs

```diff
@@ -1109,9 +1109,6 @@ fn ignore_lldb(config: &Config, line: &str) -> IgnoreDecision {
 }
 
 fn ignore_llvm(config: &Config, line: &str) -> IgnoreDecision {
-    if config.system_llvm && line.starts_with("no-system-llvm") {
```

Comment:
can `config.system_llvm` be removed?

Reply:
It's needed for min-system-llvm-version.